exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -237,33 +260,32 @@ func (b *Bitfinex) UpdateTradablePairs(forceUpdate bool) error {
 		return err
 	}
 
-	return b.UpdatePairs(currency.NewPairsFromStrings(pairs), asset.Spot, false, forceUpdate)
+	return b.UpdatePairs(currency.NewPairsFromStrings(pairs),
+		asset.Spot,
+		false,
+		forceUpdate)
```

Comment:
So `UpdateTradablePairs` is only for spot. This could be changed to
```
	for i := range b.CurrencyPairs.AssetTypes {
		pairs, err := b.FetchTradablePairs(b.CurrencyPairs.AssetTypes[i])
		if err != nil {
			return err
		}
		err = b.UpdatePairs(currency.NewPairsFromStrings(pairs),
			b.CurrencyPairs.AssetTypes[i],
			false,
			forceUpdate)
		if err != nil {
			return err
		}
	}
	return nil
```

Reply:
Epic suggestion

Reply:
Its... out of scope for the work you're doing, but its a "while you're there" :) Thanks for making that change

Reply:
Yeah, this is how we do it on exchanges which support multiple assets: https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/bitmex/bitmex_wrapper.go#L250